tests/test_messages.py

```diff
@@ -261,17 +261,19 @@ def test_call_error_representation():
 def test_call_representation():
     call = Call(unique_id="1", action=Action.Heartbeat, payload={})
 
-    assert str(call) == "<Call - unique_id=1, action=Heartbeat, payload={}>"
+    assert str(call) == (
+        f"<Call - unique_id=1, action={Action.Heartbeat}" ", payload={}>"
+    )
 
 
 def test_call_result_representation():
     call = CallResult(
         unique_id="1", action=Action.Authorize, payload={"status": "Accepted"}
     )
 
-    assert (
-        str(call) == "<CallResult - unique_id=1, action=Authorize, payload={'status': "
-        "'Accepted'}>"
+    assert str(call) == (
+        f"<CallResult - unique_id=1, action={Action.Authorize}"
```

Comment:
* For `python >= 3.11`, `Action.Authorize` returns `"Action.Authorize"`.
* For `python < 3.11`, `Action.Authorize` returns `"Authorize"`.

Reply:
Related to https://github.com/mobilityhouse/ocpp/issues/447